### PR TITLE
Raidboss: add e12s ja translation

### PR DESCRIPTION
**Pull request by MaikoTan**
https://github.com/quisquous/cactbot/blob/40997fd7785e8a5166a80e7b44ef8ef9e9e44d91/ui/raidboss/data/05-shb/raid/e12s.txt#L129

This is "Dark Water III / Eruption", right? if I am not misunderstand?

EDIT:
Some bugs in CN trans, gotta fix it tommorrow, 'cause it is already midnight here. (=_=;)

### CODE REVIEW

File: ui/raidboss/data/05-shb/raid/e12s.js [hunk 1]
```diff
@@ -49,73 +49,99 @@ const primalOutputStrings = {
...
     en: 'Out',
     de: 'Raus',
+    ja: '離れる',
+    cn: '远离',
     ko: '바깥',
   },
   '0091': {
     en: 'Intercards',
     de: 'Interkardinale Himmelsrichtungen',
+    ja: '対角',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/raid/e12s.js:73]: @r4ha Folks in CN use 対角線 to describe intercards movement when I ask them,
but I don't very clear if folks in JP also use this word?

**Review by r4ha — CHANGES_REQUESTED** [review]

File: ui/raidboss/data/05-shb/raid/e12s.js (hunk 1 above)
Comment (MaikoTan) [reply ui/raidboss/data/05-shb/raid/e12s.js:73]: http://kanatan.info/archives/25637304.html
はなつというスキル
ガルーダのパターン
image

Comment (r4ha) [reply ui/raidboss/data/05-shb/raid/e12s.js:73]: ありがとうございます！
日本のプレイヤーは"斜め"と言っているようです

File: ui/raidboss/data/05-shb/raid/e12s.js [hunk 2]
```diff
@@ -49,73 +49,99 @@ const primalOutputStrings = {
...
     en: 'Middle',
     de: 'Mitte',
+    ja: '中へ',
+    cn: '中间',
     ko: '중앙',
   },
   '008F': {
     en: 'Sides',
     de: 'Seiten',
+    ja: '両側',
```
Comment (r4ha) [inline ui/raidboss/data/05-shb/raid/e12s.js:59]: '横へ' is better here.

**Review by r4ha — CHANGES_REQUESTED** [review]

File: ui/raidboss/data/05-shb/raid/e12s.js (hunk 1 above)
Comment (MaikoTan) [reply ui/raidboss/data/05-shb/raid/e12s.js:73]: そうですか!
勉強になりました！
Thank you very much! I'll change it when I return home.

**Comment by MaikoTan** [comment]
All done!
It is safe to merge.

**Pull request merged**